deployment/Puppetfile

```diff
@@ -1,5 +1,6 @@
 forge "https://forgeapi.puppetlabs.com"
 
+mod 'garethr-docker', '5.3.0'
```

Comment:
Not sure, it looks exactly the same as: `garethr-docker`
Actually it depends on it: https://forge.puppet.com/puppetlabs/docker_platform/dependencies